sdk/tests/iot/adu/test_az_iot_adu.c

```diff
@@ -41,7 +41,8 @@ static uint8_t expected_agent_state_long_payload[]
       "0.8.0-rc1-public-preview\"},\"compatPropertyNames\":\"manufacturer,model\","
       "\"lastInstallResult\":{\"resultCode\":0,\"extendedResultCode\":1234,\"resultDetails\":"
       "\"Ok\",\"step_0\":{\"resultCode\":0,\"extendedResultCode\":1234,\"resultDetails\":\"Ok\"}},"
-      "\"state\":0,\"workflow\":{\"action\":3,\"id\":\"51552a54-765e-419f-892a-c822549b6f38\"},"
+      "\"state\":0,\"workflow\":{\"action\":3,\"id\":\"51552a54-765e-419f-892a-c822549b6f38\","
```

Comment:
Nit: we could have an UT without the retryTimestamp if that is something that the server could send today.

Reply:
Good point. Added additional as opposed to changing current